wiki/core/markdown/__init__.py

```diff
@@ -13,8 +13,7 @@
 class ArticleMarkdown(markdown.Markdown):
 
     def __init__(self, article, preview=False, *args, **kwargs):
-        kwargs = settings.MARKDOWN_KWARGS
-        kwargs['extensions'] = self.get_markdown_extensions()
+        kwargs = {'extensions': self.get_markdown_extensions()}
```

Comment:
This removes any kwargs from the caller... I'll make a second PR, thanks for reporting this!!